cmd/verify-ens/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/status-im/status-go/eth-node/bridge/geth/ens"
+	enstypes "github.com/status-im/status-go/eth-node/types/ens"
+	"go.uber.org/zap"
+)
+
+const infuraToken = "putYourTokenHere"
+const verifyENSContractAddress = "0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e"
```

Comment:
Not really used.

Reply:
WDYM? It is used in line 27

Reply:
I meant further down the stack. I didn't know that when looking up the value and later sharing it with you.